README.md

```diff
@@ -59,12 +59,18 @@ See the [config for more options](https://github.com/riemers/ansible-gitlab-runn
 Example Playbook
 ----------------
 ```yaml
-- hosts: all
-  remote_user: root
+- hosts: gitlab-runner
+  become: yes
```

Comment:
Only an example, but i see you copied the proxy part from the docs of ansible. I'll leave it in.